irishstatutebook.ie,"HTTPSConnectionPool(host='irishstatutebook.ie', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbb150af0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='irishstatutebook.ie', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbae5a940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
irisplaza.co.jp,"HTTPSConnectionPool(host='irisplaza.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbb0cf310>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='irisplaza.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbae78250>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
isa.gov.il,"HTTPSConnectionPool(host='isa.gov.il', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbae784c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='isa.gov.il', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbafe14f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
isi.ac.id,"HTTPSConnectionPool(host='isi.ac.id', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='isi.ac.id', port=80): Read timed out. (read timeout=2)"
iss.it,"HTTPSConnectionPool(host='iss.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbad04a30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='iss.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbadfc550>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
istanbulkitapcisi.com,"HTTPSConnectionPool(host='istanbulkitapcisi.com', port=443): Read timed out.","HTTPConnectionPool(host='istanbulkitapcisi.com', port=80): Read timed out. (read timeout=2)"
itau.com.uy,"HTTPSConnectionPool(host='itau.com.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbad62310>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='itau.com.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbad62e20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
its-mo.com,"HTTPSConnectionPool(host='its-mo.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbae12e80>: Failed to establish a new connection: [Errno 113] No route to host'))","HTTPConnectionPool(host='its-mo.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbad70340>: Failed to establish a new connection: [Errno 113] No route to host'))"
itsvet.com,"HTTPSConnectionPool(host='itsvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='itsvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
itu.org.il,"HTTPSConnectionPool(host='itu.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='itu.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
iu.edu.jo,"HTTPSConnectionPool(host='iu.edu.jo', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iu.edu.jo', port=443): Read timed out. (read timeout=2)"
ivivu.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
jadrolinija.hr,"HTTPSConnectionPool(host='jadrolinija.hr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9fbac7bfd0>, 'Connection to jadrolinija.hr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='jadrolinija.hr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9fbad56430>, 'Connection to jadrolinija.hr timed out. (connect timeout=2)'))"
jaf.mil.jo,"HTTPSConnectionPool(host='jaf.mil.jo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='jaf.mil.jo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
janisroze.lv,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
japanpost.jp,"HTTPSConnectionPool(host='japanpost.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbafe1e50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='japanpost.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbb0c7910>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
javerianacali.edu.co,"HTTPSConnectionPool(host='javerianacali.edu.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbadbaf40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='javerianacali.edu.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbabd36d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
jawharati.ma,"HTTPSConnectionPool(host='jawharati.ma', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='jawharati.ma', port=443): Read timed out. (read timeout=2)"
jccm.es,"HTTPSConnectionPool(host='www.jccm.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.jccm.es', port=443): Read timed out. (read timeout=2)"
jeddah.gov.sa,"HTTPSConnectionPool(host='jeddah.gov.sa', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbabcab50>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='jeddah.gov.sa', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbac5fdf0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
jelgavniekiem.lv,"HTTPConnectionPool(host='www.jelgavniekiem.lv', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.jelgavniekiem.lv', port=80): Read timed out. (read timeout=2)"
jenatadnes.com,"HTTPSConnectionPool(host='www.jenatadnes.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.jenatadnes.com', port=443): Read timed out. (read timeout=2)"
jenite.bg,"HTTPSConnectionPool(host='jenite.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='jenite.bg', port=443): Read timed out. (read timeout=2)"
jetro.go.jp,"HTTPSConnectionPool(host='jetro.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fba888670>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='jetro.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbabb27c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
jgypk.hu,"HTTPSConnectionPool(host='jgypk.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'jgypk.hu' doesn't match either of '*.mediacenter.hu', 'mediacenter.hu'"")))",Exceeded 30 redirects.
jica.go.jp,"HTTPSConnectionPool(host='jica.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fba837580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='jica.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fba8fcaf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
jma.go.jp,"HTTPSConnectionPool(host='jma.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fba8ce370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='jma.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbad886d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
jobkarov.com,"HTTPSConnectionPool(host='jobkarov.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='jobkarov.com', port=80): Read timed out. (read timeout=2)"
jobsalert.pk,"HTTPSConnectionPool(host='jobsalert.pk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='jobsalert.pk', port=443): Read timed out. (read timeout=2)"
jobscentral.com.sg,"HTTPSConnectionPool(host='jobscentral.com.sg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='jobscentral.com.sg', port=443): Read timed out. (read timeout=2)"
jobsite.co.uk,"HTTPSConnectionPool(host='www.jobsite.co.uk', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.jobsite.co.uk', port=80): Read timed out. (read timeout=2)"
jobtopgun.com,"HTTPSConnectionPool(host='jobtopgun.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.jobtopgun.com', port=443): Read timed out. (read timeout=2)"
jornaldeangola.ao,"HTTPSConnectionPool(host='jornaldeangola.ao', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='jornaldeangola.ao', port=80): Read timed out. (read timeout=2)"
joshinweb.jp,"HTTPSConnectionPool(host='joshinweb.jp', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='joshinweb.jp', port=80): Read timed out. (read timeout=2)"
joy.ne.jp,"HTTPSConnectionPool(host='joy.ne.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fba989be0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='joy.ne.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbaccf280>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
jpa.gov.my,"HTTPSConnectionPool(host='jpa.gov.my', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9fba8fc130>, 'Connection to jpa.gov.my timed out. (connect timeout=2)'))","HTTPConnectionPool(host='jpa.gov.my', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9fb9dab9d0>, 'Connection to jpa.gov.my timed out. (connect timeout=2)'))"
jpn.org,"HTTPSConnectionPool(host='jpn.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbabc76d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='jpn.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fba7464c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
jr-odekake.net,"HTTPSConnectionPool(host='jr-odekake.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fb9ee9280>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='jr-odekake.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fb9e01be0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
jreast.co.jp,"HTTPSConnectionPool(host='jreast.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fb9e01850>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='jreast.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fba895d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
js88.com,"HTTPSConnectionPool(host='js88.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9fba910bb0>, 'Connection to js88.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='js88.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9fba72f520>, 'Connection to js88.com timed out. (connect timeout=2)'))"
jsm.lt,"HTTPSConnectionPool(host='www.jsm.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.jsm.lt', port=443): Read timed out. (read timeout=2)"
jst.go.jp,"HTTPSConnectionPool(host='jst.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fb9dafd90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='jst.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fba989160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
jtb.co.jp,"HTTPSConnectionPool(host='www.jtb.co.jp', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.jtb.co.jp', port=80): Read timed out. (read timeout=2)"
jtconsulting.be,"HTTPSConnectionPool(host='jtconsulting.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fb9f2b4f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='jtconsulting.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fb9ee9790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
juduo.cc,"HTTPSConnectionPool(host='juduo.cc', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.juduo.cc', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
jurbarkosviesa.lt,"HTTPSConnectionPool(host='jurbarkosviesa.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='jurbarkosviesa.lt', port=443): Read timed out. (read timeout=2)"
juridicas.com,"HTTPSConnectionPool(host='juridicas.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'juridicas.com' doesn't match either of 'noticias.juridicas.com', 'www.noticias.juridicas.com'"")))","HTTPSConnectionPool(host='juridicas.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'juridicas.com' doesn't match either of 'noticias.juridicas.com', 'www.noticias.juridicas.com'"")))"
jurisprudencia.gob.sv,"HTTPSConnectionPool(host='jurisprudencia.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fbab632e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='jurisprudencia.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fbab63e50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
just.ro,"HTTPSConnectionPool(host='just.ro', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9fbafe1b20>, 'Connection to just.ro timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.just.ro', port=80): Read timed out. (read timeout=2)"
justice.bg,"HTTPSConnectionPool(host='justice.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='justice.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
jw.org,"HTTPSConnectionPool(host='www.jw.org', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.jw.org', port=443): Read timed out. (read timeout=2)"
kahawatungu.com,"HTTPSConnectionPool(host='kahawatungu.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'kahawatungu.com' doesn't match 'www.kahawatungu.com'"")))","HTTPSConnectionPool(host='kahawatungu.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'kahawatungu.com' doesn't match 'www.kahawatungu.com'"")))"
kalkulatorkalorii.net,"HTTPSConnectionPool(host='kalkulatorkalorii.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='kalkulatorkalorii.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
kalmp3indir.com,"HTTPSConnectionPool(host='kalmp3indir.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='kalmp3indir.com', port=443): Read timed out. (read timeout=2)"
kalyanamitra.org,"HTTPSConnectionPool(host='kalyanamitra.org', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='kalyanamitra.org', port=80): Read timed out. (read timeout=2)"
kam.si,"HTTPSConnectionPool(host='kam.si', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='kam.si', port=80): Read timed out. (read timeout=2)"
kamnapivo.sk,"HTTPSConnectionPool(host='kamnapivo.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='kamnapivo.sk', port=80): Read timed out. (read timeout=2)"
kamsdetmi.sk,"HTTPSConnectionPool(host='kamsdetmi.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'kamsdetmi.sk' doesn't match either of 'pobytovetabory.sk', 'www.pobytovetabory.sk'"")))","HTTPConnectionPool(host='kamsdetmi.sk', port=80): Read timed out. (read timeout=2)"
kaposvarmost.hu,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
kar.nic.in,"HTTPSConnectionPool(host='kar.nic.in', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fba9523a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='kar.nic.in', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fba85c850>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
karpol.lt,"HTTPSConnectionPool(host='www.karpol.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.karpol.lt', port=443): Read timed out. (read timeout=2)"
kathimerini.com.cy,"HTTPSConnectionPool(host='www.kathimerini.com.cy', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.kathimerini.com.cy', port=443): Read timed out. (read timeout=2)"
katoliska-cerkev.si,"HTTPSConnectionPool(host='katoliska-cerkev.si', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='katoliska-cerkev.si', port=443): Read timed out. (read timeout=2)"
kaunoklinikos.lt,"HTTPSConnectionPool(host='kaunoklinikos.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='kaunoklinikos.lt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f9fba377b50>, 'Connection to kaunoklinikos.lt timed out. (connect timeout=2)'))"
kbc-rijeka.hr,"HTTPSConnectionPool(host='kbc-rijeka.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'kbc-rijeka.hr' doesn't match either of '*.croadria.com', 'croadria.com'"")))","HTTPConnectionPool(host='kbc-rijeka.hr', port=80): Read timed out. (read timeout=2)"
kbc-zagreb.hr,"HTTPSConnectionPool(host='kbc-zagreb.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9fba31b7c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='kbc-zagreb.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9fba4cef10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
kcb.vn,"HTTPSConnectionPool(host='kcb.vn', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='kcb.vn', port=80): Read timed out. (read timeout=2)"
